### ISSUE TITLE
Auto-upload of Media (photo, video) does not work

### ISSUE THREAD

**Issue by Chatter35** [nextcloud/android#13773]
### ⚠️ Before posting ⚠️

- [x] This is a **bug**, not a question or an enhancement.
- [x] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [x] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Switch on Auto Upload for Media
Take a photo
Wait some minutes until a photo/video should have been uploaded form Android to Nextcloud.
Now: While photos/videos appear in the settings overview for Auto Upload - the media are NOT loaded to the Nexcloud server.

Manual work around:
Create an own Auto Upload folder and select the identical Android source and Nexcloud target folder as in the original setting. Now, media get uploaded to Nextcloud. 

Yet, RAW (.dng) photos and jps appear in the some "Camara" folder - but RAW photos are not loaded in "RAW" folder.

### Expected behaviour

When media Auto upload is activated they should be loaded to Nextcloud.
Ideally, jpg appears in "Camara" folder and dng in "RAW" folder.

### Actual behaviour

No auto upload of media (photos and videos)

### Android version

14

### Device brand and model

Google Pixel 8 Pro

### Stock or custom OS?

Custom (explain in "additional information")

### Nextcloud android app version

Stable version 3.30.1 from GitHub

### Nextcloud server version

28.0.10

### Using a reverse proxy?

I don't know

### Additional information

GrapheneOS

**Comment by joshtrichards** [comment]
> Switch on Auto Upload for Media

Can you provide the precise settings you used since there are numerous variations?

> but RAW photos are not loaded in "RAW" folder.

To clarify: Are you just having a problem with RAW/.dng photos?

**Comment by Chatter35** [comment]
> > Switch on Auto Upload for Media
> 
> Can you provide the precise settings you used since there are numerous variations?
> 
> > but RAW photos are not loaded in "RAW" folder.
> 
> To clarify: Are you just having a problem with RAW/.dng photos?

To clarify: Photos are not uploaded automatically although the option has been switched on. This pertains any format, i.e. jpg and dng. (Auto-upload of jpg and dng to separate folders has worked in the past - but I would see this as a 'nice to have'.)

**Comment by joshtrichards** [comment]
Great but we still the config details requested. 

For example:

It sounds like the breakage is when using an auto detected folder only since it sounds like configuring a *custom folder* was a workaround?

What options are set on the relevant auto detected auto upload profile(s)? E.g. *Also upload existing files*, etc.

From the sounds of it this is a change in behavior for this device? When did it stop working as expected? Or is this new setup on this device?

**Comment by Chatter35** [comment]
I have just tried again:

Taken two photos with Google Camara on GrapheneOS (w/o any Google Play Services): 1. jpg only, 2 RAW+jpg (i.e. three files in total). Surprisingly, the photos have been auto-uploaded instantly (from local folder „Pixel 8 Pro - DCIM - Camara“ to the folder „Nextcloud - AutoUpload - Camara“). So, it looks like auto-upload works fine again.
Also, photos I have received via Singal Messenger have been auto-uploaded (into a Nextcloud folder „Pictures“).

Notably, I did not change any setting. But if I remember correctly, there was a recent update of GrapheneOS.

(Ideally, I would have wished that RAW photos could be directed via AutoUpload into another Nextcloud folder than „Camara“, which has worked in the past - but this might have been a function of the Galerie app by GrapheneOS.)

(One thing that still does NOT work: Internal 2-way-sync. When clicking on this link a white screen without any folder appears.)

Thank you for having followed up on this. For me the issue seems resolved now.

Cheers.
